l3kernel/l3fp-aux.dtx

```diff
@@ -731,9 +731,9 @@
 % following trick splits it into two blocks of $4$ digits, padding
 % with zeros on the left.
 % \begin{verbatim}
-%   \cs_new:Npn \pack:NNNNNw #1 #2#3#4#5 #6; { {#2#3#4#5} {#6} }
-%   \exp_after:wN \pack:NNNNNw
-%     \__fp_int_value:w \__fp_int_eval:w 1 0000 0000 + #1 ;
+%   \cs_new:Npn \__fp_pack:NNNNNw #1 #2#3#4#5 #6 \__fp_sep: { {#2#3#4#5} {#6} }
+%   \exp_after:wN \__fp_pack:NNNNNw
+%     \__fp_int_value:w \__fp_int_eval:w 1 0000 0000 + #1 \__fp_sep:
```

Comment:
This code snippet and the surrounding paragraphs do not adhere to the current implementation and usage of `\__fp_pack:NNNNNw`.

It's not adjusted in this PR, as I'm not familiar with fp implementation and it's not relevant to this PR.